pkg/safeguards/manifestresults.go

```diff
@@ -0,0 +1,108 @@
+package safeguards
+
+import (
+	"context"
+	"embed"
+	"fmt"
+
+	api "github.com/open-policy-agent/gatekeeper/v3/apis"
+	log "github.com/sirupsen/logrus"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+)
+
+// Globals
+var s = runtime.NewScheme()
```

Comment:
where is the scheme actually used? intellisense isn't picking it up for me

Reply:
seems fine to me then since it's only used for that and you control what's in the constraint templates